apps/auth/src/app/components/sign-x-via-google/sign-x-via-google.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { TuiButtonModule } from '@taiga-ui/core';
+
+@Component({
+  selector: 'oak-sign-x-via-google',
+  standalone: true,
+  imports: [
+    CommonModule,
+    TuiButtonModule
+  ],
+  templateUrl: './sign-x-via-google.component.html',
```

Comment:
this button does not need a separate component.html file

Let this be a thumb rule...if the file contains about 10-20 lines of HTML just place it in the template section of the component file...same with the scss files

Reply:
Also inform Adriana about this